site/layouts/partials/callout-warning-data-bs-title-vs-title.md

```diff
@@ -0,0 +1 @@
+Note that `title` attribute can be used instead of `data-bs-title` and will be removed and rendered directly in the Popper element.
```

Comment:
How's something like this sound? Mostly want to double check that the `title` is replaced with `data-bs-title`.

```suggestion
Feel free to use either `title` or `data-bs-title` in your HTML. When `title` is used, Popper will replace it automatically with `data-bs-title` when the element is rendered.
```

Reply:
Thanks that's better explained like that. I commit your suggestion.